src/core/unit.h

```diff
@@ -167,6 +167,9 @@ struct Unit {
          * process SIGCHLD for */
         Set *pids;
 
+        Set *deferred_dropin_dependencies;
+        Set *deferred_dropin_units;
```

Comment:
could add a comment to these explaining what kind of object they point to? I don't think that's obvious in this case